e2e/wdio.conf.js

```diff
@@ -79,9 +80,11 @@ export const config = {
   // ensure we are running `tauri-driver` before the session starts so that we can proxy the webdriver requests
   beforeSession: async () => {
     // tauriDriver = spawn(tauriDriverPath, [], {
-    tauriDriver = spawn("pnpm", ["tauri-driver"], {
-      stdio: [null, process.stdout, process.stderr],
-    });
+    tauriDriver = spawn(
+      path.resolve(os.homedir(), ".cargo", "bin", "tauri-driver"),
+      [],
+      { stdio: [null, process.stdout, process.stderr] }
+    );
```

Comment:
```suggestion
    tauriDriver = spawn("pnpm", ["tauri-driver"], {
      stdio: [null, process.stdout, process.stderr],
      shell: true,
    });
```

i think this should work